GoogleMaps-SwiftUI/GoogleMaps-SwiftUI/GoogleMapView/GoogleMapView.swift

```diff
@@ -0,0 +1,106 @@
+// Copyright 2024 Google LLC. All rights reserved.
+//
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under
+// the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+// ANY KIND, either express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+import SwiftUI
+import GoogleMaps
+
+/// A SwiftUI wrapper for GMSMapView that displays a map with optional markers and configurable map type
+struct GoogleMapView: UIViewRepresentable {
+   /// Binding to map options that can be updated from parent view
+   @Binding var options: GMSMapViewOptions
```

Comment:
Consider naming this bound set `properties: GoogleMapViewProperties` per comment in GoogleMaps_SwiftUIApp.swift

Reply:
The @Binding parameter for the GMSMapViewOptions variable has been removed. Since GoogleMapView works as a wrapper for GMSMapView we are still required to follow the same conventions. This includes using GMSMapViewOptions for map initialization.